### ISSUE TITLE
What is ui/js/min.js?

### ISSUE THREAD

**Issue by pfalcon** [dhobsd/castty#31]
Re: https://github.com/dhobsd/castty/issues/4, etc., can it be confirmed that min.js contains all of:

*  jQuery
* rangeslider.js
* xterm.js

(from the README) in the minimize form? What versions of the components were used? How that file can be (re)produced?

**Comment by dhobsd** [comment]
Yes, this is basically what's there. I'm not familiar with best practices for vendoring things for UI stuff. It'd be nice to have a release process to make packages for things. I don't recall what versions were used, but I'd be open to someone else doing this work if it also means that you can still have a working thing just checking out the repo.
